homeassistant/components/flo/config_flow.py

```diff
@@ -28,7 +28,7 @@ async def validate_input(hass: core.HomeAssistant, data):
         raise CannotConnect from request_error
 
     user_info = await api.user.get_info()
-    a_location_id = user_info["locations"][0]["id"]
+    a_location_id = user_info["locations"][1]["id"]
```

Comment:
Might worth adding a comment explaining why the first location is ignored.

Would it be more reliable if you get each location in this array before making a call on which one should be used as part of the user setup?

Reply:
@frenck @kirill578 I like the suggestion, however looking at the latest version aioflo they no longer surface "nickname" for the locations so having a selection page for that would be not very user friendly. 

This integration doesn't actually use the location IDfor anything except for naming the integration. The better fix would be to eliminate the location_id from the config flow and use the username to name the integration instead. I'm working on a fix now and will submit a pull request.